config/anlaytics-settings.js

```diff
@@ -0,0 +1,4 @@
+exports.analyticsIds = {
+  production: 'UA-55446531-10',
+  development: 'UA-55446531-19'
+};
```

Comment:
I recommend exporting two keys, here, so we can easily destructure them when we import.  Also, could we name them something non-generic? Just so it's trivial to find them when we need to maintain.

`prodAnalyticsId` and `devAnalyticsId` for example